### PR TITLE
Milestone 1: Capstone-project

### PR DESCRIPTION
**Pull request by ArthurGC**
# First Capstone Project | Web Design Course
:beginner: In this first milestone I:

- [ ] Set up my repository and tools.
- [ ] Choose the topic for my website. (Web Development Course - Web Design Course)
- [ ] Choose images, text, icons, fonts. (I made my own images and icons. I used the fonts from the guideline provided by Microverse)
- [ ] Create the 2 pages for mobile. (Home and About pages)
- [ ] Adapt the 2 pages to desktop.
- [ ] Implement the user interactions: link, mobile menu.
- [ ] Implement the section "Featured speakers" with dynamic HTML.
- [ ] Record a video describing my project.
- [ ] Create a good README and pull request description.
- [ ] Deploy the project.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                    <a href="#" class="link">Join for Free</a>
+                </li>
+            </ul>
+            <i class="fas fa-bars hamb"></i>
+        </nav>
+    </header>
+    <main>
+        <section class="headline">
+            <p class="headline__greet">"Hello World!"</p>
+            <h1 class="headline__title">Web Design for Everybody: Basics of Web Development and Coding</h1>
```
Comment (Shaher-11) [inline about.html:42]: Kindly style the heading as the given design below
right1

File: about.html [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+            <div class="headline__dsc">
+                <p>This Specialization covers how to write syntactically correct HTML5 and CSS3, and how to create
+                    interactive web experiences with JavaScript. Mastering this range of technologies will allow you to
+                    develop high quality web sites that, work seamlessly on mobile, tablet, and large screen browsers
+                    accessible. During the capstone you will develop a professional-quality web portfolio demonstrating
+                    your growth as a web developer and your knowledge of accessible web design. This will include your
+                    ability to design and implement a responsive site that utilizes tools to create a site that is
+                    accessible to a wide audience, including those with visual, audial, physical, and cognitive
+                    impairments.</p>
+            </div>
```
Comment (Shaher-11) [inline about.html:52]: decrease the width of the div as shown below
right2

File: about.html [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+                <li class="past-courses__li">
+                    <a href="#"><img src="public/past-course-1.png" alt="first course image">
+                    <p class="past-courses__p"><span>2019</span>CodeClub in Peru</p></a>
+                </li>
+                <li class="past-courses__li">
+                    <a href="#"><img src="public/past-course-2.png" alt="second course image">
+                    <p class="past-courses__p"><span>2020</span>CodeClub in Mexico</p></a>
+                </li>
+            </ul>
+        </section>
```
Comment (Shaher-11) [inline about.html]: Reduce the size of the text here and the socila icons 

#### Original Design
right3

File: index.html [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+                </li>
+                <li class="card">
+                    <div class="presenter__image">
+                        <img src="public/presenter-3-rodnae.svg" alt="presenter photo">
+                    </div>
+                    <div class="presenter__dsc">
+                        <h3>Stella Martin</h3>
+                        <p class="dsc-1">Front-End Developer from France</p>
+                        <p class="dsc-2">Professional with 8 years of experience in Web Design.</p>
+                    </div>
```
Comment (Shaher-11) [inline index.html]: the image is not dispayed here
wrong3

File: index.html [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+                    <p class="program__p">Start the course of JavaScript and Responsive Design</p>
+                </li>
+                <li>
+                    <img src="public/Icon-1.svg" alt="lecture icon">
+                    <h3>Closing Party</h3>
+                    <p class="program__p">Closing the course and exchange of opinions about it.</p>
+                </li>
+            </ul>
+        </section>
+        <section class="presenter">
```
Comment (Shaher-11) [inline index.html]: The section should be designed like the below screenshot and take advantage of css flex
right4

File: index.html [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+    <main>
+        <section class="headline">
+            <p class="headline__greet">"Hello World!"</p>
+            <h1 class="headline__title">Web Design for Everybody: Basics of Web Development and Coding</h1>
+            <div class="headline__dsc">
+                <p>Learn to Design and Create Websites. Build a responsive and accessible web portfolio using HTML5, CSS3, and JavaScript.</p>
+            </div>
+            <p class="headline__date">2021.07.21(TUE) ~ 22(THU)</p>
+        </section>
+        <section class="program">
```
Comment (Shaher-11) [inline index.html]: This section should look like the screenshot below on the desktop
right5

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @ArthurGC,

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Add the loom video link to the readme file.
- [ ] You should follow these [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) guidelines, including: Colors, Typography: font face, size, and weight, Layout: composition and space between elements.
- [ ] Check the comments below:point_down:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi dear student, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

star_wars_almost_there


## Aspects to improve ♻️
_Check the comments under the review._

- It looks like you could improve the size a little bit, why we don't try to make little these parts on your project.
- [ ] The `Offered by` and `footer` section needs to make size changes to be like the original design.
image

- [ ] This part is big as needed, please, fix this 😄
image

- Fix these two details and you'll be on the other side.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 5 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @ArthurGC , 

You did great work with making all the changes requested by previous TSEs.
Kudos for using SCSS :+1:

Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

[Optional]
- [ ] Top dark part of the menu is missing on your project. Looks like previous TSEs missed this. This will not affect the status of your project but if you want to make your project similar to the design template, you can add this part also.
Screenshot 2021-07-01 at 02-19-59 CC Global Summit 2015


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**